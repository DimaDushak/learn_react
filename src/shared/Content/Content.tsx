import React from 'react';
import styles from './content.css';

interface Props {
    children: React.ReactNode;
}

export default function Content({ children }: Props): JSX.Element {
    return <main>{children}</main>;
}
