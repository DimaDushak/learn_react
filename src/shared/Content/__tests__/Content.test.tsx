import { render } from '@testing-library/react';
import React from 'react';
import Content from '../Content';

test('should render Content (snapshot)', () => {
    const { container } = render(<Content>Контент</Content>);

    expect(container.firstChild).toMatchSnapshot();
});
