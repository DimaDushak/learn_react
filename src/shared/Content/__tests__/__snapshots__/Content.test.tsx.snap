// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Content (snapshot) 1`] = `
<main>
  Контент
</main>
`;
